pkg/virt-controller/services/rendervolumes.go

```diff
@@ -25,14 +25,15 @@ import (
 type VolumeRendererOption func(renderer *VolumeRenderer) error
 
 type VolumeRenderer struct {
-	containerDiskDir string
-	ephemeralDiskDir string
-	virtShareDir     string
-	namespace        string
-	vmiVolumes       []v1.Volume
-	podVolumes       []k8sv1.Volume
-	podVolumeMounts  []k8sv1.VolumeMount
-	volumeDevices    []k8sv1.VolumeDevice
+	containerDiskDir      string
+	ephemeralDiskDir      string
+	virtShareDir          string
+	namespace             string
+	vmiVolumes            []v1.Volume
+	podVolumes            []k8sv1.Volume
+	podVolumeMounts       []k8sv1.VolumeMount
+	sharedFilesystemPaths []string
```

Comment:
Nit: maybe `sharedFilesystemsPath` would be a bit better?

Reply:
`Path` not plural? are you sure?

Reply:
No, you're right about `Paths` (plural), I just happened to make a typo :)

I was suggesting `Filesystems` (plural) instead of `Filesystem` (singular). It just sounds a bit better to me. But I'm not a native speaker so feel free to disregard this.